docs/demo.md

```diff
@@ -37,7 +37,7 @@ assert project["createdDateTime"] == "2022-09-12T00:00:00Z"
 
 ### POST
 
-Here we are looking at accepting a body as input and passing it to the service. This function is not actually in the QnA authoring cadl, but is more to demonstrate what an input body would look like
+Here we are looking at accepting a body as input and passing it to the service. This function is not actually in the authoring cadl, but is more to demonstrate what an input body would look like
```

Comment:
what does `QnA` mean?

Reply:
Good question, it's the name of the service we're generating cadl from, we either call it QnA authoring or just authoring, so both work